Jimp UTIF dependency breaking elasticbeanstalk/AWS deploys

# Expected Behavior

The application runs without errors

# Current Behavior

When I tried to update JIMP to 0.3.5 it's break my AWS deploy once UTIF uses jpgjs from github, not from npm, probably AWS doesn't accept this dependency type

## Steps to Reproduce

1. Update JIMP to 0.3.5
2. Deploy to elasticbeanstalk/AWS

## Context

-   Jimp Version: 0.3.5
-   Operating System: Linux
-   Node version: 8.9.3

## Failure Logs
I received this message:

Error: Cannot find module 'jpgjs'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at /var/app/current/node_modules/utif/UTIF.js:10:74
    at Object.<anonymous> (/var/app/current/node_modules/utif/UTIF.js:911:3)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/app/current/node_modules/*jimp*/dist/utils/image-bitmap.js:20:36)

--- a/packages/type-tiff/package.json
+++ b/packages/type-tiff/package.json
@@ -17,7 +17,7 @@
   "license": "MIT",
   "dependencies": {
     "core-js": "^2.5.7",
-    "utif": "^2.0.0"
+    "utif": "^2.0.1"
   },
   "peerDependencies": {
     "@jimp/custom": ">=0.3.5"

